localisation/00_wc_vanilla_replacements.csv

```diff
@@ -451,3 +451,32 @@ EVTDSC_RIP_21019_INTRIGUE_A;The old woman introduces herself as [From.GetTitledF
+school_tao_1_c;Shangqing, to infuse my spirit with wisdom!;;;;;;;;;;;;;x
+school_zhengyi;Follower of Zhengyi School;;;;;;;;;;;;;x
+school_quanzhen;Follower of Quanzhen School;;;;;;;;;;;;;x
+school_shangqing;Follower of Shangqing School;;;;;;;;;;;;;x
+choose_school_taoist;Choose [Root.Religion.GetGroupName] School;;;;;;;;;;;;;x
+EVTDESC_MNM_8023;A large group of sauroks block our way. The local ruler is a man that has taken it upon himself to relieve pilgrims, merchants or other travelers of some of their wealth in order to allow them passage. Turning back is not an option, either we pay or we will have to fight these men.;;;;;;;;;;;;;x
+EVTDESC_MNM_8029;You come across a thriving city in the middle of a dry upland. The secret of their prosperity seems to be huge water tanks, built and maintained by the local ruler. A separate reservoir is also under construction to allow the city to grow even bigger. You stop to admire this feat of engineering and cannot help wondering how much a similar construction could benefit [Root.Capital.GetName].;;;;;;;;;;;;;x
+indian_pilgrimage_desc;Go on a Pilgrimage to one of the holy places of Pandaria.;;;;;;;;;;;;;x
+EVTDESC_MNM_8061;This place is invigorating to the mind. Being able to engage with so many great minds of religion and theology has stimulated my understanding and willingness to explore the finer points of the faith. Hours and even days are lost in conversation with priests and monks about the deeper meaning of various passages in [Root.Religion.GetScriptureName].;;;;;;;;;;;;;x
+EVTDESC_MNM_8062;Purification and Enlightenment have never before felt as tangible as in this place. Pilgrims are circumambulating shrines and monks and gurus meditate together. I join those in worship that for thousands of years have attended this place and others like it. This is a profound experience unlike anything I have ever felt before.;;;;;;;;;;;;;x
+EVTDESC_MNM_8063;Many pilgrims have stories of the experiences from their journeys and the great enlightenment or purification achieved in at the destination. I expected much more than I have found here. The place is certainly full of monks, priests and holy men but neither circumambulation, meditation or cleansing has left me with anything but an empty feeling of doubt.;;;;;;;;;;;;;x
+EVTDESC_MNM_8064;Eager to engage other in discussion I rushed right ahead to the centers of learning. I can tell the ideas and thoughts I have had over the years have made an impression on others here. Even great minds, scholars and monks alike seem taken by my reasoning and my home-grown theological arguments.;;;;;;;;;;;;;x
+EVTDESC_MNM_8068;A holy place such as this has always attracted mystics and monks. After completing my cleansing bath in the Ganges and started to climb back up the ghats I met an old wrinkled man who bestowed his blessing upon me. Ever since I have felt more powerful and invigorated.;;;;;;;;;;;;;x
```

Comment:
The Ganges is a river in India, so should probably be replaced with something more appropriate. Same with the Ghats (a mountain range). 
```suggestion
EVTDESC_MNM_8068;A holy place such as this has always attracted mystics and monks. After completing my cleansing bath in the Ganges and started to climb back up the ghats I met an old wrinkled man who bestowed his blessing upon me. Ever since I have felt more powerful and invigorated.;;;;;;;;;;;;;x
```

Reply:
changed to Yan-Zhe and Kun-Lai